src/librustc_typeck/diagnostics.rs

```diff
@@ -4146,8 +4102,6 @@ register_diagnostics! {
 //  E0068,
 //  E0085,
 //  E0086,
-    E0103, // @GuillaumeGomez: I was unable to get this error, try your best!
-    E0104,
```

Comment:
@arielb1 commented them out.

Also instead of adding whole documentation as a comment, I've added `// E0101` and `// E0102` to `register_diagnostics!` call. 

If you have both the commented documentation and a `register_diagnostics!` call for an error code, `tidy` complains. Even if they're only comments.